scipy/signal/filter_design.py

```diff
@@ -375,6 +549,325 @@ def zpk2tf(z, p, k):
     return b, a
 
 
+def tf2sos(b, a, pairing='simple'):
+    """
+    Return second-order sections from transfer function representation
+
+    Parameters
+    ----------
+    b : array_like
+        Numerator polynomial coefficients.
+    a : array_like
+        Denominator polynomial coefficients.
+    pairing : {'simple', 'keep_odd'}
+        The pairing method to use. See `zpk2sos`.
```

Comment:
elaborate a little, like "the method used to combine poles and zeros into sections"?